index.fr.md

```diff
@@ -0,0 +1,111 @@
+    des capteurs de pression
+-   appareils utilisés avec des paramètres différents comme les écrans lisibles en plein soleil (transflectifs)
+-   et bien d'autres
+
+## W3C WAI traite l'accessibilité mobile {#covered}
+
+**Les standards d'accessibilité formulés par la WAI traite de l'accessibilité mobile** :
+
+-   **Les Règles pour l'accessibilité des contenus Web** ([WCAG](/standards-guidelines/wcag/)) traitent des pages et applications web, incluant le contenu à destination des appareils mobiles.
+    -   Pour apprendre comment WCAG 2.0 s'applique au contenu web pour mobile, aux applications web pour mobile, aux applications natives et aux applications hybrides utilisant des composantes web, voir [Accessibilité Mobile : Comment les WCAG 2.0 et autres règlements de la WAI du W3C s'appliquent au Mobile](http://www.w3.org/TR/mobile-accessibility-mapping/).
+    -   Une ressource plus générale qui inclut également les applications mobiles est : [WCAG2ICT : Appliquer les WCAG 2.0 à l'information non web et aux technologies de communication](http://www.w3.org/TR/wcag2ict/).
+    -   WCAG 2.1, les règles publiées en Juin 2018 contiennent de nouvelles exigences ("critères de succès") traitant de l'accessibilité mobile.  Elles sont présentées dans : [[Nouveautés dans les WCAG 2.1]](/standards-guidelines/wcag/new-in-21/).
+-   **Les Règles pour l'accessibilité des agents utilisateurs** ([UAAG](/standards-guidelines/uaag/)) traitent des navigateurs web ainsi que d'autres agents utilisateurs, y compris les navigateurs sur appareils mobiles.
+    -   Pour consulter des exemples sur les bénéfices accordés aux personnes souffrant de handicaps lors de la navigation web sur un appareil mobile, voir les [Exemples d'accessibilité Mobile dans le contexte des UAAG](http://www.w3.org/TR/IMPLEMENTING-UAAG20/mobile).
+    -   Pour ceux qui souhaitent comprendre plus en détail les problèmes rencontrés, voir [Appliquer les UAAG aux téléphones mobiles](http://www.w3.org/WAI/UA/work/wiki/Applying_UAAG_to_Mobile_Phones).
+-   **Les Règles pour l'accessibilité des outils d'édition** ([ATAG](/standards-guidelines/atag/)) traitent des outils utilisés pour créer des pages web et applications, incluant la création sur appareils mobiles.
+-   **[WAI-ARIA](/standards-guidelines/aria/)** (Applications Internet Riches et Accessibles) décrit la façon de rendre le contenu web plus accessible, spécifiquement pour les contenus dynamiques et les contrôles d'interfaces avancées. Cela s'applique aux applications web et à l'accès de sites web avec un appareil mobile.
+
+ 
+
+**Le W3C traite l'accessibilité mobile.** La WAI s'assure que les technologies centrales du W3C prennent en charge l'accessibilité, y compris celles qui sont essentielles pour le Web mobile. L'ensemble du travail effectué par le W3C est examiné par le [Groupe de travail Architectures de Plateforme Accessible (APA)](https://www.w3.org/WAI/APA/).
```

Comment:
- "W3C addresses mobile accessibility" : see my remark about "traite l'accessibilité mobile" above.
- The name of the APA group is tricky to translate. I guess "Architecture de plateformes accessibles" is easier to understand in French and closer to the original name, but I think we should discuss this with @slhenry and the rest of the group. What do you think ?

Reply:
- replaced with "aborde"
- I think your suggestion is better than mine. Replacing it for now but we can still discuss with @slhenry indeed.

Reply:
For the APA WP, I've just opened an issue for the glossary (and changed my eye a bit) https://github.com/w3c/translation-glossaries/issues/23

Reply:
Commented on the ISSUE.